src/types/request.rs

```diff
@@ -0,0 +1,104 @@
+use actix_web::{web::Bytes, HttpRequest};
+use dashmap::DashMap;
+use pyo3::{prelude::*, types::PyDict};
+use std::collections::HashMap;
+
+use super::{check_body_type, get_body_from_pyobject, Url};
```

Comment:
one minor nit - can you replace `super` with an absolute path? I am not a big fan of relative imports.

Reply:
Done :)